packages/editor/src/plugins/fill-in-the-blanks-exercise/blank-renderer.tsx

```diff
@@ -30,7 +30,8 @@ export function BlankRenderer({ element }: BlankRendererProps) {
 
     // Focus editor when the blank is removed
     return () => {
-      ReactEditor.focus(editor)
+      // TODO: uncommented this breaks the preview mode somehow, investigate
+      // ReactEditor.focus(editor)
```

Comment:
@hejtful do you have an idea why this could happen? 
when I activate the preview this line breaks the editor (you could test it in this branch).

@CodingDive this might have got you as well over in https://github.com/serlo/frontend/pull/3263

Reply:
merged [your fix](https://github.com/serlo/frontend/pull/3266)